src/pages/payment-methods/index.js

```diff
@@ -32,18 +43,38 @@ const Th = styled.th`
         left: -5px; /* required */
         background-color: var(--color-white);
         z-index: 2;
-
+        width: 128px;
+        padding: 16px 0 16px 16px;
+    }
+    :nth-child(7) {
         p {
             text-align: center;
         }
     }
+    :last-child {
+        width: 50px;
+    }
+`
+const SectionTopContainer = styled(SectionContainer)`
+    @media ${device.tabletL} {
+        padding: 48px 16px 40px;
+    }
+`
+const SectionContentContainer = styled(SectionContainer)`
+    @media ${device.tabletL} {
+        padding: 24px 0 40px;
+    }
+`
+const TopContainer = styled(Container)`
+    @media ${device.tabletL} {
+        width: 100%;
+    }
```

Comment:
We can pass all these tabletL styles in as props for e.g. `<SectionContainer tabletL={{ p: "48px 16px 40px" }}></SectionContainer>`.  You can refer to the base component for the responsive styles config for ref.

Reply:
Thanks for the feedback Sean. I have changed the TopContainer style based on this. However sectionContainer has some conflicts in style and waiting to get fixed in this card - https://redmine.deriv.cloud/issues/51208#Section_Container_component_enhancement